src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -298,6 +298,10 @@ private function displayDeprecations(array $groups, Configuration $configuration
             return $b->count() - $a->count();
         };
 
+        $outputIsCli = !$configuration->shouldWriteToLogFile();
+        // TODO: fix this; we need to detect here if the `simple-phpunit` command was run with `-v` option
```

Comment:
We shouldn't build this for the simple-phpunit script specifically. We must do this following PHPunit's own executable (especially as we're recommending not using simple-phpunit for applications)